gke-dask/notebooks/xgboost-gpu-hpo-job-parallel-k8s.ipynb

```diff
@@ -18,283 +18,20 @@
    "id": "19c0c861-9357-48b1-a317-a9df3deb2319",
```

Comment:
<p>I think we should either block this PR until this is in a released version, or open an issue to track coming back and doing that once the Optuna release happens.</p><p>I don't mind which.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
@jacobtomlinson Let's merge this PR for now, since it makes the notebook so much more user-friendly. (No more MySQL!) I'll create an issue as a reminder to follow up.

Reply:
Awesome thanks @hcho3!